tests/test_synthetic.py

```diff
@@ -67,3 +67,21 @@ def test_unknown_emoji():
     # characters. The mangled text passes through as is.
     not_emoji = 'dos burritos: \U0003f32f\U0003f32f'.encode('utf-8').decode('windows-1251')
     eq_(fix_encoding(not_emoji), not_emoji)
+
+    # The range we accept as emoji has gotten larger. Let's make sure we can
+    # decode the futuristic emoji U+1F960, which will probably be a picture of
+    # a fortune cookie in Unicode 10.0:
+    emoji_text = "\U0001f960 I see emoji in your future"
+    emojibake = emoji_text.encode('utf-8').decode('windows-1252')
+    eq_(fix_encoding(emojibake), emoji_text)
+
+    # We believe enough in the future of this codepoint that we'll even
+    # recognize it with a mangled byte A0
+    emojibake = emojibake.replace('\xa0', ' ')
+    eq_(fix_encoding(emojibake), emoji_text)
+
+    # Increment the first byte to get a very similar test case, but a
+    # codepoint that will definitely not exist anytime soon. In this case,
+    # we consider the existing text, "ñŸ¥\xa0", to be more probable.
+    not_emoji = "\U0005f960 I see mojibake in your present".encode('utf-8').decode('windows-1252')
+    eq_(fix_encoding(not_emoji), not_emoji)
```

Comment:
Oh, I meant specifically the `\U0003f32f` part.  (It doesn't seem like a big deal anyway, though.)